From: Eric Alexander <eric.alexander@gmail.com>
Subject: Fix dis51: FTBFS: pass2.c:186:4: error: format not a string literal and no format arguments [-Werror=format-security]
 During a rebuild of all packages in sid, your package failed to build on
 amd64. This patch fixes the FTBFS.
Bug-Debian: http://bugs.debian.org/643374
Index: dis51-0.5/pass2.c
===================================================================
--- dis51-0.5.orig/pass2.c
+++ dis51-0.5/pass2.c
@@ -183,7 +183,7 @@ static int dis_inst2(FILE *ofile, const
 			fprintf(ofile, "DB 85h  ; illegal opcode");
 		case 1:
 				/* no operands */
-			fprintf(ofile, mnemonic[opcode]);
+			fprintf(ofile, "%s", mnemonic[opcode]);
 			break;
 		case 2:
 				/* one immediate operand */
